qa/suites/rgw/multifs/omap_limit/10.yaml

```diff
@@ -0,0 +1,5 @@
+overrides:
+  ceph:
+    conf:
+      osd:
+        osd_max_omap_entries_per_request: 10
```

Comment:
<pre>
int cls_cxx_map_get_vals(cls_method_context_t hctx, const string &start_obj,
			 const string &filter_prefix, uint64_t max_to_get,
			 map<string, bufferlist> *vals)
</pre>
I think this needs a new out arg line bool *done, and all the callers need to get fixed accordingly.